devtools/conda.recipe/meta.yaml

```diff
@@ -2,18 +2,18 @@
 
 package:
   name: weldx
-  version: {{ data["version"] }}
+  version: {{ data.get("version") }}
```

Comment:
I changed it, because locally I received an error like this (with a very recent version of conda-build):
```INFO:conda_build.metadata:Attempting to finalize metadata for weldx
Error: Failed to render jinja template in /home/marscher/sources/weldx/devtools/conda.recipe/meta.yaml:
'data' is undefined
```